### PR TITLE
[release-v1.10] HTTP/2 disable patch from upstream

### PR DESCRIPTION
**Pull request by matzew**
Basicially this PR just runs:
```
./hack/update-deps.sh --upgrade --release 1.10
```

### CODE REVIEW

File: vendor/knative.dev/pkg/webhook/stats_reporter.go [hunk 1]
```diff
@@ -106,8 +106,9 @@ func (r *reporter) ReportAdmissionRequest(req *admissionv1.AdmissionRequest, res
 		return err
 	}
 
-	// TODO skonto: fix latency histogram
-	metrics.Record(ctx, requestCountM.M(1))
+	metrics.RecordBatch(ctx, requestCountM.M(1),
+		// Convert time.Duration in nanoseconds to milliseconds
+		responseTimeInMsecM.M(float64(d.Milliseconds())))
```
Comment (pierDipi) [inline vendor/knative.dev/pkg/webhook/stats_reporter.go:111]: There is a reverted patch here

Comment (matzew) [reply vendor/knative.dev/pkg/webhook/stats_reporter.go:111]: yeah :-)

**Comment by matzew** [comment]
@pierDipi updated, and all green

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**